components/molecule/imageEditor/README.md

```diff
@@ -21,10 +21,12 @@ return (
   <MoleculeImageEditor 
     cropLabelIcon={<CropIcon>}
     cropLabelText="Crop"
+    helpText="Some indication helpful for the user"
     image={image}
     onChange={handleChange}
     rotateLabelIcon={<RotateIcon>}
     rotateLabelText="Rotate"
+    useRowLayout={false}
```

Comment:
don't name this `"use"` 
It is preferred to name it `is[whatever]` or `has[whatever]` because in React the prefix `use` is used for hooks

Reply:
good point, agree. Also being a `boolean` value it should be, as you say, `is[whatever]` or `has[whatever]`.